.circleci/config.yml

```diff
@@ -93,9 +93,8 @@ executors:
   ubuntu:
     <<: *working_directory
     machine:
-      image: ubuntu-2004:2022.10.1
+      image: ubuntu-2204:2024.01.2
```

Comment:
is there any reason why or this was just a nice thing to do?

Reply:
The older image came with a `go` version that was too old to build after these updates